File: C/fizzbuzz1.c
#include <stdio.h>

#define LIMIT 100

int main()
{
	int i;

	for (i = 1; i <= LIMIT; i++) {
		if (i % 15 == 0) {
			printf("FizzBuzz");
		} else if (i % 3 == 0) {
			printf("Fizz");
		} else if (i % 5 == 0) {
			printf("Buzz");
		} else {
			printf("%d", i);
		}
		if (i < LIMIT) {
			printf(", ");
		}
	}
	printf("\n");
}

File: Zig/fizzbuzz1.zig
const std = @import("std");

const limit: u8 = 100;

pub fn main() !void {
    const stdout = std.io.getStdOut().writer();
    var i: u8 = 1;

    while (i <= limit) : (i += 1) {
        if (i % 15 == 0) {
            try stdout.print("{s}", .{"FizzBuzz"});
        } else if (i % 3 == 0) {
            try stdout.print("{s}", .{"Fizz"});
        } else if (i % 5 == 0) {
            try stdout.print("{s}", .{"Buzz"});
        } else {
            try stdout.print("{d}", .{i});
        }
        if (i < limit) try stdout.print(", ", .{});
    }
    try stdout.print("\n", .{});
}
